dawn/renderer/reducers/info.js

```diff
@@ -28,7 +28,6 @@ const info = (state = {}, action) => {
     case 'UPDATE_STATUS':
       return {
         ...state,
-        isRunningCode: (action.status.value === 1),
```

Comment:
Yes. It is currently modified as otherwise, we wouldn't be able to show peripheral display.